player/lua/osc.lua

```diff
@@ -2072,6 +2085,16 @@ local function osc_init()
     bind_mouse_buttons("volume")
 
 
+    -- custom buttons
+    for i = 1, 99 do
```

Comment:
why not do `while user_opts["custom_button_" .. i .. "_content"]` and increment i on each loop? Stop of first not populated button.